Cargo.toml

```diff
@@ -20,6 +20,9 @@ features = ["winit"]
 [dependencies]
 stretch = "0.2"
 twox-hash = "1.5"
+chrono = "0.4.9"
+log = "0.4.8"
+simplelog = "0.7.2"
```

Comment:
I think [`env_logger`](https://docs.rs/env_logger/0.6.2/env_logger/) is a simpler alternative for the example. We should also add it as a development dependency instead.

Reply:
Hmm but `env_logger` can't write a log-file 😬

Reply:
You are right! However, I think it is not a necessary feature for the example and we should probably try to keep it as simple and straightforward as possible.